source/src/drivers/cs_Serial.cpp

```diff
@@ -225,12 +233,9 @@ extern "C" void UART0_IRQHandler(void) {
 		if (_readCallback != NULL) {
 			_readCallback(readByte);
 		}
-
 	}
-
 	else if (NRF_UART0->EVENTS_RXTO && nrf_uart_int_enable_check(NRF_UART0, NRF_UART_INT_MASK_RXTO)) {
-//		nrf_uart_event_clear(NRF_UART0, NRF_UART_EVENT_RXTO);
-		// TODO: init and start UART.
+		// TODO: clear events, init and start UART.
```

Comment:
Lots of commented stuff is removed as according to the style guide. I've another patch that allows to actually set the baudrate because it is hardcoded here.